polymer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "entrypoint": "index.html",
```

Comment:
Ah, please don't add a service worker. This is a standalone element, and a service worker is a user feature, not an element feature.